.travis.yml

```diff
@@ -34,15 +34,7 @@ global:
 addons:
   apt:
     packages:
-      # - dosbox
```

Comment:
At some point, we'll need to figure it out :) It would speed up the build a lot, I guess.